frontend/src/metabase/public/components/EmbedModal/SelectEmbedTypePane/SelectEmbedTypePane.tsx

```diff
@@ -171,13 +201,13 @@ export function SelectEmbedTypePane({
             >{t`Settings`}</Link>
           </Text>
         )}
-        <a
+        <ExternalLink
```

Comment:
This fixes the link [opening in the same tab unexpectedly](https://www.notion.so/metabase/Streamline-embed-flow-when-embedding-features-are-off-13869354c90180b4bc02ca29ebf61ad2?pvs=4#14969354c90180b0872ef52aa27317e5).